packages/common/src/services/audius-backend/AudiusBackend.ts

```diff
@@ -1146,7 +1146,7 @@ export const audiusBackend = ({
 
   async function repostTrack(trackId: ID) {
     try {
-      return audiusLibs.Track.addTrackRepost(trackId)
+      return audiusLibs.EntityManager.repostTrack(trackId)
```

Comment:
Do we want to feature flag this?

Reply:
i think after we've dialed up playlists to 100% we should be good on making changes without a feature flag! i had asked @raymondjacobson the same thing when making this change.